internal/app.go

```diff
@@ -164,18 +160,18 @@ func (a *App) Bootstrap() error {
 		}
 	}
 
-	osVersions := []string{}
-	if a.CurrentOSVersion != "" {
-		osVersions = append(osVersions, a.CurrentOSVersion)
-	}
-	if a.NextOSVersion != "" {
-		osVersions = append(osVersions, a.NextOSVersion)
-	}
-
-	// TODO(cdc) When we have the list of available packages for an OS version,
-	// pick the best one.
-	if err := a.InstallAddon("docker", a.DockerVersions[0], a.OSChannel, osVersions, MinimumRemoteDocker); err != nil {
-		return err
+	if !a.Conf.SkipTorcxSetup {
```

Comment:
Minor nit: this is probably more readable by removing the NOT and swapping the then-else branches order.